include/morphio/enums.h

```diff
@@ -33,6 +33,7 @@ enum Warning
     UNDEFINED,
     MITOCHONDRIA_WRITE_NOT_SUPPORTED,
     WRITE_NO_SOMA,
+    WRITE_EMPTY_MORPHOLOGY,
```

Comment:
I would append the new `WRITE_EMPTY_MORPHOLOGY` enum value at the end to prevent shifting all existing identifiers.